changelogs/unreleased/317-add-web-ui-section-to-the-configuration-file-iso5.yml

```diff
@@ -0,0 +1,8 @@
+---
+description: The 'dashboard' and 'web-console' sections are being deprecated. The 'web-ui' section should now be used instead for configuration options related to web interfaces.
+change-type: minor
+issue-nr: 317
+issue-repo: inmanta-ui
+destination-branches: [iso5]
```

Comment:
This is an invalid PR. The base branch of this PR is `master`, but the changelog entry indicates that it should be merged into `iso5` only.